docs/security_arch.md

```diff
@@ -0,0 +1,54 @@
+# Security Architecture
+
+Corpora's infrastructure is managed by the [single-cell-infra repo](https://github.com/chanzuckerberg/single-cell-infra)
+which was provided by the Shared Infrastructure team in order to enable streamlined creation of secure cloud
+environments. This document depicts the application of the template environment in Corpora to illustrate its inner
+complexities. Ideally, general documentation on the `cloud-env` environment from Shared Infrastructure should supplement
```

Comment:
Can you take out Ideally, it make it sounds wishy washy. Does the documentation exist or doesn't it? Who should we probe to get it?

Reply:
can you make cloud-env link to cloud-env in single-cell-infra or somewhere applicable.